src/plots/mapbox/layout_attributes.js

```diff
@@ -48,10 +48,16 @@ var attrs = module.exports = overrideAll({
         dflt: constants.styleValueDflt,
         role: 'style',
         description: [
-            'Sets the Mapbox map style.',
-            'Either input one of the default Mapbox style names or the URL to a custom style',
-            'or a valid Mapbox style JSON.',
-            'From OpenStreetMap raster tiles, use *open-street-map*.'
+            'Sets the Mapbox base map style.',
+            'Base map styles are rendered below all traces and layout layers.',
+            'Either input one of the default Mapbox style names:', constants.styleValuesMapbox, '.',
```

Comment:
```suggestion
            'Either input one of the default Mapbox style names: ', constants.styleValuesMapbox, '.',
```

Reply:
In the same vein, `constants.styleValuesMapbox.join(', ')` to ensure we have a whitespace after each comma.

Reply:
`['Possible values are:', ['a','b','c']].join(' ')` -> "Possible values are:  a,b,c"

`['Possible values are:',['a','b','c'].join(', ')].join(' ')` -> "Possible values are: a, b, c"

Reply:
done in https://github.com/plotly/plotly.js/pull/4035/commits/ca762e96d9f8ea77966cfe4918c292e48661f3b0 

Thanks!